test/quantization/pt2e/test_x86inductor_quantizer.py

```diff
@@ -1225,26 +1276,28 @@ def test_linear_binary(self):
                     node_occurrence = {
                         # Only one 1 q-dq for input of the linear
                         # No q-dq for extra input node of add
-                        torch.ops.quantized_decomposed.quantize_per_tensor.default: 1,
-                        torch.ops.quantized_decomposed.dequantize_per_tensor.default: 1,
+                        quantize_per_tensor_op: 1,
+                        dequantize_per_tensor_op: 1,
                         # quantize_per_channel for weights are const propagated
                         torch.ops.quantized_decomposed.quantize_per_channel.default: 0,
                         torch.ops.quantized_decomposed.dequantize_per_channel.default: 1,
                     }
                 else:
+                    # convert_pt2e disables duplicate dequant for dynamic quant
+                    num_dequant = 1 if is_dynamic else 2
```

Comment:
but why things are different for static quant? Can we dedup dequant too?

Reply:
Meta decided to duplicate dq for static quant but not for dynamic quant: https://github.com/pytorch/pytorch/pull/115337#discussion_r1433318916 The discussion is long. The conclusion is not to duplicate dq for dynamic quant

Reply:
I only saw the discussion on the dynamic quant. Did I miss anything? Do you have the specific link to explain why we dup in the static quant but do not dup in the dynamic quant? I just don't understand why we make things inconsistent here. cc @jerryzh168

Reply:
The discussion was about dynamic quantization only. The difference of behavior between static and dynamic quantization is due to the design XNNPACK workflow. Need comments from @jerryzh168 for more details.

Reply:
there were some discussions here before: https://github.com/pytorch/pytorch/pull/115337#discussion_r1446496919

Reply:
I think dup for static quant should probably be removed as well, this should be done by the lowering pass